src/app/inventory/dim-item-info.ts

```diff
@@ -179,45 +146,10 @@ export function cleanInfos(stores: DimStore[]): ThunkResult {
   };
 }
 
-/**
- * Load infos in the old way we used to save them.
- */
-export async function getOldInfos(
-  account: DestinyAccount,
-  data: Readonly<DimData>
-): Promise<Readonly<ItemInfos>> {
-  let oldKey = `dimItemInfo-m${account.membershipId}-p${account.originalPlatformType}-d${account.destinyVersion}`;
-
-  let infos = data[oldKey];
-  if (!infos) {
-    // Steam players may have originally been Blizzard players
-    if (account.originalPlatformType === BungieMembershipType.TigerSteam) {
-      oldKey = `dimItemInfo-m${account.membershipId}-p${BungieMembershipType.TigerBlizzard}-d${account.destinyVersion}`;
-      infos = data[oldKey];
-    }
-  }
-
-  if (infos) {
-    ga('send', 'event', 'Item Tagging', 'Old Version');
-    // Convert to new format
-    const newInfos = _.mapKeys(infos, (_, k) => k.split('-')[1]);
-    await SyncService.remove(oldKey);
-    return newInfos;
-  }
-
-  return {};
-}
-
-export function getTag(
-  item: DimItem,
-  itemInfos: InventoryState['itemInfos']
-): TagValue | undefined {
-  return itemInfos[item.id] && itemInfos[item.id].tag;
+export function getTag(item: DimItem, itemInfos: ItemInfos): TagValue | undefined {
+  return itemInfos[item.id]?.tag || undefined;
```

Comment:
no technical reason for `||undefined` unless `tag` was somehow both a falsey & invalid value

Reply:
I think the issue here was that tag could be `null`.